src/main/java/seedu/address/model/person/Person.java

```diff
@@ -99,8 +99,12 @@ public boolean isSamePerson(Person otherPerson) {
             return true;
         }
 
-        return otherPerson != null
-                && otherPerson.getName().equals(getName());
+        boolean isSameName = otherPerson != null
+                && otherPerson.getName().getLowerCaseName().equals(getName().getLowerCaseName());
```

Comment:
Since the equals method in Name already checks if lowerCaseName is the same, is it possible to use the equals method in `Name` instead of manually checking if lowerCaseName for both persons are the same?

Reply:
Good catch, I agree that it's better. Just modified to use `Name#equals` instead.